### PR TITLE
Panel design

### PR DESCRIPTION
**Pull request by ARKhoshghalb**
Changes to panel design
Separated views and templates

### CODE REVIEW

File: apps/accounts/templates/accounts/oldpanel/battle_history.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                            {% if match.single_matches.first.log %}
+                                <a class="ui label"
+                                   href="{% url 'game:game viewer' %}?log_url={{ match.single_matches.first.log.name }}&first_team={{ match.part1.depend.team.name|urlencode }}&second_team={{ match.part2.depend.team.name|urlencode }}">{% trans 'View' %}</a>
+                            {% endif %}
+                        </td>
+                    {% elif match.part2.depend == participation %}
+                        <td>{{ match.part1.depend.team.name }}</td>
+                        <td>{% if match.single_matches.first.log %}{{ match.score2 }}{% endif %}</td>
+                        <td>{% if match.single_matches.first.log %}{{ match.score1 }}{% endif %}</td>
+                        <td></td>
```
Comment (altostratous) [inline apps/accounts/templates/accounts/oldpanel/battle_history.html]: Please complete this like the other branch of the if-statement.

File: apps/accounts/templates/accounts/panel/battle_history.html [hunk 2]
```diff
@@ -17,7 +18,7 @@ <h2 style="margin-top: 20px;">{% trans 'Battle history' %}</h2>
             <tbody>
             {% for match in battle_history %}
                 <tr>
-                    <td>{{ match.time }}</td>
+                    <td>{{ match.time.date|jalali }} - {{ match.time|time }}</td>
```
Comment (altostratous) [inline apps/accounts/templates/accounts/panel/battle_history.html]: Please check the site language

Comment (altostratous) [reply apps/accounts/templates/accounts/panel/battle_history.html]: It is not important

File: aic_site/settings/base.py [hunk 3]
```diff
@@ -124,7 +124,7 @@
 )
 LANGUAGE_CODE = 'fa'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Asia/Tehran'
```
[the remark this answers is not in the document]
Comment (ARKhoshghalb) [reply aic_site/settings/base.py]: data in always saved in UTC in database. it's just the way we show it and I still don't know why we are writing in english

**Pull request merged**